graphql/types/Profile.graphql

```diff
@@ -20,7 +24,10 @@ input ProfileInput {
   homePhone: String,
   businessPhone: String,
   document: String,
-  email: String,
```

Comment:
User can't modify its own email, otherwise, it will lose its link with the Master Data.